src/plugins/analysis/users_and_passwords/code/password_file_analyzer.py

```diff
@@ -63,9 +59,9 @@ def _generate_analysis_entry(self, passwd_entries):
                 if entry[1][0] == ord('$'):
                     result[key]['password-hash'] = entry[1].decode(encoding='utf_8', errors='replace')
                     cracked_pw = self._crack_hash(entry, result, key)
-                    result[key]['cracked'] = True if cracked_pw else False
-            except Exception as e:
-                logging.error('Invalid Format: {} - {}'.format(sys.exc_info()[0].__name__, e))
+                    result[key]['cracked'] = bool(cracked_pw)
+            except Exception as excep:
+                logging.error('Invalid Format: {} - {}'.format(sys.exc_info()[0].__name__, excep))
```

Comment:
This would be much slimmer and you can drop the `sys` import:
```python
except Exception:
    logging.error('Invalid Format:', exc_info=True)
```
Also: Why can't we have a less general exception?

Reply:
```python
except (IndexError, AttributeError, TypeError):
```
should do I guess.

Reply:
> This would be much slimmer and you can drop the `sys` import:
> 
> ```python
> except Exception:
>     logging.error('Invalid Format:', exc_info=True)
> ```
We should have a discussion about our "default" error logging and we should document it somewhere